css/style.css

```diff
@@ -26,6 +28,24 @@
   margin: 10px auto;
 }
 
+h3 {
+  font-size: 38px;
+  color: #272a31;
+}
+
+footer p {
+  margin-bottom: 12px;
+}
+
+.third h3 {
+  font-size: 25px;
+  color: #272a31;
+}
+
+.third p {
+  margin-bottom: 30px;
+}
+
 .main-program a {
   text-decoration: underline;
 }
```

Comment:
- [ OPTIONAL] Kindly make sure to add the `border` effect when the user hover any box of the main-program section

![main](https://user-images.githubusercontent.com/79058364/169443599-e58401a3-358c-4dab-ae4f-b51f3638bbf9.png)